x/crosschain/keeper/cctx.go

```diff
@@ -48,15 +59,42 @@ func (k Keeper) SetCctxAndNonceToCctxAndInboundHashToCctx(
 		in.CctxIndex = append(in.CctxIndex, cctx.Index)
 	}
 	k.SetInboundHashToCctx(ctx, in)
+}
 
-	if cctx.CctxStatus.Status == types.CctxStatus_Aborted && cctx.InboundParams.CoinType == coin.CoinType_Zeta {
+// updateZetaAccounting updates the zeta accounting with the amount of zeta that was locked in an aborted cctx
+func (k Keeper) updateZetaAccounting(
```

Comment:
You mean `ZetaAccounting` , 
It was initially added as part of a recommendation of an audit to track the zeta supply on the chain. 
It was ment to be used with the `ZetaSupplyChecker` to add checks for the supply. In case we are not planning to use this design for the supply checks , we can remove this ( In a separate PR )